docs/THEORY.md

```
@@ -7,6 +7,8 @@
 - [Factor Clusters](#-factor-clusters)
 - [Design Decision: ODE is not Campus](#-design-decision-ode-is-not-campus)
 - [Emergent Properties](#-emergent-properties)
+- [Institutional Quality](#-institutional-quality)
+- [Grading & Outcome Classification](#-grading--outcome-classification)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)